data/routers/project-routers.js

```diff
@@ -0,0 +1,159 @@
+const express = require("express");
+const router = express.Router();
+const projects = require("../helpers/projectModel");
+
+router.get("/api/projects", (req, res) => {
+  projects
+    .get(req.body.id)
+    .then(post => {
+      if (post === null) {
+        res.status(404).json({ message: "There is no project with this id" });
+      } else {
+        res.status(200).json(post);
+      }
+    })
+    .catch(error => {
+      res
+        .status(500)
+        .json({ message: "An error occurred collecting the data" });
+    });
+});
+
+router.get("api/projects/:id/actions", (req, res) => {
+  const { id } = re.params;
+  projects
+    .get(req.params.id)
+    .then(post => {
+      if (post === null) {
+        res.status(404).json({ message: "There isn't a project with this id" });
+      } else {
+        projects
+          .getProjectActions(req.params.id)
+          .then(actions => {
+            if (actions.length === 0) {
+              res
+                .status(200)
+                .json({ message: "There isn't a project with this id" });
+            } else {
+              res.status(200).json(actions);
+            }
+          })
+          .catch(error => {
+            res
+              .status(500)
+              .json({ message: "There was an error collecting action data" });
```

Comment:
It is always good to include the just created post as part of the response to a post request
```suggestion
              .json({
          message: "The Post has been successfully created.",
          post
        });
```